identities/models.py

```diff
@@ -9,6 +9,8 @@
 from django.core.exceptions import ValidationError
 from rest_hooks.signals import raw_hook_event
 
+ADDRESS_TYPES = ['msisdn', 'email']
```

Comment:
I'm not super comfortable with this since we don't restrict the address types anywhere else in the Identity store. But I guess since it's only for the metrics it's probably fine. Please put it in the settings file instead of here though, that way it'll be easily changeable if we do want something different.